### PR TITLE
Focus the metrics markdown files

### PR DESCRIPTION
**Pull request by germonprez**
Ultimately, I just removed the notes below the description of the metrics. My thought is that if we are going to offer these metrics as something to tool towards, we should reduce the noise that we represent. 

If we want to retain the notes that lead into the description of the metrics, that's easy enough to do.

### CODE REVIEW

**Comment by GeorgLink** [comment]
@germonprez 
Thank you for the pull request. 
Looks like a great idea - I will post a detailed review later.

**Comment by germonprez** [comment]
Rechecking my notes, we need to include toxicity in Growth-Maturity-Decline

File: 3_Risk.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Qualified Committers | Contributions over time and what components they commit t
 User Dependency | Number of users who are aware that they depend on the software over time
```
Comment (rpaik) [inline 3_Risk.md]: Not sure what this is saying/measuring.  Are we saying it's a bad thing for a code to be used by a lot of users?

File: 3_Risk.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ Qualified Committers | Contributions over time and what components they commit t
 User Dependency | Number of users who are aware that they depend on the software over time
 Paid developers | Number of paid developers in community over time
```
Comment (rpaik) [inline 3_Risk.md]: Paid by who?  There are a lot of developers who are paid by her/his employer to contribute to an open source project (e.g. OpenStack, Kubernetes, etc.)

File: 3_Risk.md [hunk 3]
```diff
@@ -2,71 +2,24 @@
 
 Activity Metric | Description
 --- | ---
-Contributor Importance | Percentage of commits by individual contributors
-Qualified Committers | Contributions over time and what components they commit to
-User Dependency | Number of users who are aware that they depend on the software
-Paid developers | Number of paid developers in community
+Contributor Importance | Percentage of commits by individual contributors over time
```
Comment (rpaik) [inline 3_Risk.md]: Should it be Percentage of commits by individual contributors from certain organizations?  I think this sentence need clarity as all contributions are from individuals....

File: 3_Risk.md (hunk 1 above)
Comment (GeorgLink) [reply 3_Risk.md]: This came out of the OSSNA focus group. In the case of OpenSSH, few users knew that they depended on the software. In the group we talked about a potential action that may result from this information, for example communicating that the bug existed, giving it a name "heartbleed", and educating the public in layman's terms. If people do not know they depend on a software, they may not bother updating or reporting issues. - we are not saying anything about usage, only about awareness.

File: 3_Risk.md (hunk 2 above)
Comment (GeorgLink) [reply 3_Risk.md]: Paid to work on the project vs. paid but not specifically to work on the project. - I don't know how to best differentiate the two.

File: 3_Risk.md (hunk 3 above)
Comment (germonprez) [reply 3_Risk.md]: Yea, that should be clarified in the definition. Thanks.

File: 1_Diversity-Participation.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Metric: Diversity and Participation
+
+Activity Metric | Description
+--- | ---
+New Contributors vs Maintainers | Ratio of new contributors to maintainers over time
```
Comment (jgbarah) [inline 1_Diversity-Participation.md]: Is there any difference between "contributors" and "maintainers"? If so, I think we would need a definition. If not, I suggest to use the same word everywhere (likely, "contributors"), to avoid confusion.

File: 1_Diversity-Participation.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Metric: Diversity and Participation
+
+Activity Metric | Description
+--- | ---
+New Contributors vs Maintainers | Ratio of new contributors to maintainers over time
+Contributor Demographics | Gender, age, location, education, and skills over time
+New Contributor Corporate Affiliations | New corporate affiliations over time
+New Contributions | Percentage of contributions (patches, pull requests, etc.) from new contributors vs all accepted contributions over time
+Accepted Code Contributions | Percentage of new contributor code versus total code over time
+Path to Maintainership | Path to maintainership communicated
```
Comment (jgbarah) [inline 1_Diversity-Participation.md]: Could we have an example of this, or maybe a short explanation?

File: 2_Growth-Maturity-Decline.md [hunk 6]
```diff
@@ -3,75 +3,26 @@
 
  Activity Metric | Description
  --- | ---
- Tickets Closed | Total number of tickets closed
- Time to Close | Time from bug opened to bug closed
- Time to Merge | Time from submission to merge or abandon
- Review Efficiency | Number of merged patches / Number of abandoned patches
+ Tickets Closed / Tickets Opened | Total number of tickets closed per tickets opened over time
```
Comment (jgbarah) [inline 2_Growth-Maturity-Decline.md]: This is, for any given period (eg, a month), the number of tickets closed during that period (whenever they were opened) by the number of tickets opened during that month, right?

File: 2_Growth-Maturity-Decline.md [hunk 7]
```diff
@@ -3,75 +3,26 @@
...
-The two-hour session sparked great conversations but could not address all aspects.
-We thank everyone who participated.
+**Tooling:**
+The activity metrics are intended to be a starting point for community health related tooling. It is expected that the activity metrics will evolve based on the ability (or inability) of tooling to successfully implement the activity metrics. 
 
-Below is the metric->signal->action chain for *Growth-Maturity-Decline*.
+**Background:**
+The activity metics have been identified based on workshops at the Open Source Leadership and the Open Source Summit North America. In addition, the activity metrics are based on active CHAOSS mailing list conversations. The activity metrics listed here are the result of compiling the discussions to data. We thank everyone who participated.
 
 **How to contribute:**
```
Comment (jgbarah) [inline 2_Growth-Maturity-Decline.md]: Maybe it would be better to have a link to a "contributing.md" page, which would have the text in all these pages about contribution? That way we have a single point for maintaining info on contributions.

File: LICENSE [hunk 8]
```diff
@@ -2,20 +2,19 @@ MIT License
```
Comment (jgbarah) [inline LICENSE]: Since these are basically descriptions, more than code, wouldn't it better to use the CC license? (just wondering, please ignore if you already thought on this and decided this license was the way to go).

File: 1_Diversity-Participation.md (hunk 4 above)
Comment (GeorgLink) [reply 1_Diversity-Participation.md]: How about:
- Contributor = Anyone who provides a patch, pull request, wiki change, or other contribution.
- Maintainer = Contributor who has commit rights to the main branch and can merge contributions from others.

File: 1_Diversity-Participation.md (hunk 5 above)
Comment (GeorgLink) [reply 1_Diversity-Participation.md]: Path to Maintainership = A governance document that describes the criteria and process by which a contributor can be promoted to maintainer (i.e. be granted commit rights).

File: 2_Growth-Maturity-Decline.md (hunk 6 above)
Comment (GeorgLink) [reply 2_Growth-Maturity-Decline.md]: Yes, the time period can be specified by the user of the system.

File: LICENSE (hunk 8 above)
Comment (GeorgLink) [reply LICENSE]: We considered several licenses, including CC and discussed them with Mike Dolan and Kate Stewart from the Linux Foundation before the formation of CHAOSS. Our charter reads:
8.2.3. All contributions to implementation-agnostic metrics and standards, including associated scripts, SQL statements, and documentation, will be received and made available under the MIT License (https://opensource.org/licenses/MIT).

File: 2_Growth-Maturity-Decline.md (hunk 7 above)
Comment (germonprez) [reply 2_Growth-Maturity-Decline.md]: Good idea. The reason that I included the background was to help explicate that these metrics weren't just made up :) That we have been spending time -- and will continue to spend time on what the metrics can be.

**Comment by germonprez** [comment]
Should we accept these changes and merge? Then we can talk about #30. I think it would be great to get these out there in V.001 prior to Prague.

**Comment by GeorgLink** [comment]
I agree it would be good to merge them before tomorrow morning.

@germonprez  will you make changes based on the reviews before we merge?
- [ ] you wanted to include "toxicity"
- [ ] @rpaik suggested clarification for "Contributor Importance"
- [ ] @rpaik suggested "Path to maintainership published"
- [ ] @rpaik suggested "number of maintainers added over a time period"
- [ ] @rpaik suggested clarifying " Percentage of commits by individual contributors from certain organizations"
- [ ] @jgbarah asked for a definition of maintainer and contributor
- [ ] @jgbarah suggested a CONTRIBUTING.md file

File: 1_Diversity-Participation.md (hunk 4 above)
Comment (jgbarah) [reply 1_Diversity-Participation.md]: It is not always easy to know who has those rights... And  much less to know that over time (for comparing with the past, for example).

File: 1_Diversity-Participation.md (hunk 5 above)
Comment (jgbarah) [reply 1_Diversity-Participation.md]: Thanks for the example. Maybe the metric is therefore a binary metric, something like "Existence of a public document detailing the path to mantainership"?

File: LICENSE (hunk 8 above)
Comment (jgbarah) [reply LICENSE]: OK, no problem then. I just didn't realized the wording of MIT license during those discussions.

Comment (GeorgLink) [reply LICENSE]: no worries :-)

File: 1_Diversity-Participation.md (hunk 5 above)
Comment (GeorgLink) [reply 1_Diversity-Participation.md]: That is a better description indeed.

File: 1_Diversity-Participation.md (hunk 4 above)
Comment (GeorgLink) [reply 1_Diversity-Participation.md]: I agree that this information may not be easily known for third-party projects. It may be more easily known if you are involved in the project. Thus, the metric might have value for assessing internal projects. We also wanted to capture metrics that are meaningful and then hear from the software committee if can be implemented or needs unavailable data sources.

**Comment by GeorgLink** [comment]
@germonprez thank you for the pull request and following up on requests and comments.

@rpaik and @jgbarah thank you for reviewing the changes and providing feedback.
